R/Learner.R

```diff
@@ -566,7 +575,10 @@ Learner = R6Class("Learner",
       private$.encapsulate = insert_named(default, rhs)
 
       if (is.null(private$.fallback)) {
-        self$fallback = lrn(mlr_reflections$learner_fallback[[self$task_type]], predict_type = self$predict_type)
+        fallback_id = mlr_reflections$learner_fallback[[self$task_type]]
+        if (!is.null(fallback_id)) {
+          self$fallback = lrn(mlr_reflections$learner_fallback[[self$task_type]], predict_type = self$predict_type)
```

Comment:
> This line assumes that the fallback default fallback learner supports all possible predict types. For classification and regression (once https://github.com/mlr-org/mlr3/pull/1125 is merged) this should be fine. But what about e.g. survival?

We check this now. See https://github.com/mlr-org/mlr3/pull/1151

> Also I think we should not require the is.null(fallback_id) check at runtime. We should just make sure that all the extension packages properly implement this.

It will stay that way for the time being so that we can easily get to CRAN.